### PR TITLE
Gardena: Added sensor commands

### PR DESCRIPTION
**Pull request by gerrieg**

### CODE REVIEW

**Comment by kaikreuzer** [comment]
Instead of adding further channels, wouldn't it be much more logical to simply implement the REFRESH command on the existing channels? I was already missing that feature (after seeing that the Gardena app supports this).

**Comment by gerrieg** [comment]
yes of course, good idea, i will change it

File: addons/binding/org.openhab.binding.gardena/README.md [hunk 1]
```diff
@@ -74,6 +74,28 @@ In the items file, you can link items to channels of your Things:
 Number Battery_Level "Battery [%d %%]" {channel="gardena:mower:home:myMower:battery#level"}
 ```
 
+## Sensor refresh
+You can send a REFRESH command to items linked to these Sensor channels:
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.gardena/README.md]: please add an empty line here

File: addons/binding/org.openhab.binding.gardena/README.md [hunk 2]
```diff
@@ -74,6 +74,28 @@ In the items file, you can link items to channels of your Things:
 Number Battery_Level "Battery [%d %%]" {channel="gardena:mower:home:myMower:battery#level"}
 ```
 
+## Sensor refresh
+You can send a REFRESH command to items linked to these Sensor channels: 
+* ambient_temperature#temperature
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.gardena/README.md]: add another empty line here

**Pull request merged**